packages/compiler/lib/decorators.tsp

```diff
@@ -337,6 +337,9 @@ extern dec projectedName(target: unknown, targetName: string, projectedName: str
  */
 extern dec discriminator(target: object | Union, propertyName: string);
 
+alias KnownDateFormat = "rfc1123" | "rfc7231" | "unixTimeStamp";
+extern dec dateFormat(target: zonedDateTime | ModelProperty, format: KnownDateFormat | string);
```

Comment:
Does `@dateFormat` make the most sense as the name and not `@dateSerializationFormat`. It is different from `@format` on a string which actually dictate how the string is formatted

Reply:
I like shorter names, but dateSerializationFormat is more accurate